unit-11/redo-main.js

```diff
@@ -0,0 +1,126 @@
+// Exercise 1: Repeats the elements of the array
+// Solution 1: use 'push'
+const getRepeatingArray = (character, number) => {
+  const result = [];
+  for (let i = 1; i <= number; i++) {
+    result.push(character);
+  }
+  return result;
+};
+
+// Solution 2: use 'splice'
+const showRepeatingArray = (character, number) => {
+  const result = [];
+  for (let i = 1; i <= number; i++) {
+    result.splice(i, 0, character);
+  }
+  return result;
+};
+
+// Exercise 2: Reverse array (don't use reverse)
+const getReverseArray = arr => {
+  const newArr = [];
+  for (let i = arr.length - 1; i >= 0; i--) {
+    newArr.push(arr[i]);
+  }
+  return newArr;
+};
+
+// Exercise 3: Delete the wrong element (null, undefined, 0, ' ', false)
+const getTrueValues = arr => {
+  return arr.filter(Boolean);
+};
+
+// Exercise 4:
+const converArraytToObject = data => {
+  const Obj = {};
```

Comment:
```suggestion
  const obj = {};
```

Ai cho viết hoa tên biến thế này